project/project.yml

```diff
@@ -0,0 +1,78 @@
+title: "Biaffine Parser"
```

Comment:
```suggestion
title: "Transition-based Parser"
```

Reply:
Changed the title to something more descriptive 👍 .